Data section edits (#1173)
ACCESS-Hive/access-hive.github.io: docs/model_evaluation/data/data_basics.md

### CF conventions and CMOR standards

> Most ACCESS netCDF files follow the **[Climate and Forecast (CF) metadata conventions](https://zenodo.org/records/17801666)**. CF conventions provide standardised ways to describe:

Reviewer: The website https://cfconventions.org/ seems to be active again. 
Maybe it's worth reverting this as it might be more appropriate?

Author: ```suggestion
Most ACCESS netCDF files follow the **[Climate and Forecast (CF) metadata conventions](https://cfconventions.org)**. CF conventions provide standardised ways to describe:
```

<!-- https://cfconventions.org server error April2026-->

- coordinates and grids  
- physical quantities and units  
- time and calendar definitions  

These conventions make datasets easier for humans to understand and machines to interpret. Many analysis and visualisation tools (such as [Xarray](https://xarray.dev)) rely on CF conventions to automatically recognise coordinates, apply units correctly, and handle data consistently across different models and datasets.

> The term 'CMORised' data may also be used to describe climate data standards. This refers to the use of Climate Model Output Rewriter (CMOR) which is a program writen by PCMDI (Program for Climate Model Diagnosis and Intercomparison) to apply CMIP (Coupled Model Intercomparison Project) standards to model outputs which can be submitted to [CMIP](https://wcrp-cmip.org/). CMOR standards are stricter than CF conventions which can allow for the multi-model analysis in the CMIP phases and between the generations. 

Reviewer: ```suggestion
The term _CMORised_ data may also be used to describe climate data standards. This refers to the use of Climate Model Output Rewriter (CMOR) which is a program writen by PCMDI (Program for Climate Model Diagnosis and Intercomparison) to apply CMIP (Coupled Model Intercomparison Project) standards to model outputs which can be submitted to [CMIP](https://wcrp-cmip.org/). CMOR standards are stricter than CF conventions, allowing consistent multi-model analysis both within and across CMIP generations.
```

> ACCESS-NRI supports CMORisation of ACCESS models with ACCESS-MOPPy.

Reviewer: Do we have a link for ACCESS-MOPPy? 
Even a link to the repo if nothing else is available at the moment
